api/src/main/java/javax/json/bind/JsonbConfig.java

```diff
@@ -252,7 +252,7 @@ public final JsonbConfig withStrictIJSON(final Boolean enabled) {
     /**
      * Property used to specify custom naming strategy.
      *
-     * Configures value of {@code JSONB_PROPERTY_NAMING_STRATEGY} property.
+     * Configures value of {@link #PROPERTY_NAMING_STRATEGY} property.
```

Comment:
I guess this change is OK, but there are also other places in this config which should be changed because of consistency.
For example: {@ code STRICT_IJSON} etc.

Or is there a reason why to change it only in this case?

Reply:
good catch @Verdent. I've updated my PR and also fixed javadoc in the rest of the file

Reply:
@aguibert great :-)